fala/templates/adviser/search.html

```diff
@@ -7,70 +7,88 @@
 {% import "macros/element.html" as Element %}
 {% import "macros/forms.html" as Form %}
 
-{% block search_form %}
-  <form action="" class="laa-fala__search" novalidate>
-    <div
-      class="govuk-form-group
-        {% if (form.errors and form.errors['__all__']) %}
-          govuk-form-group--error
-        {% endif %}
-        "
-      id="fala_questions">
+{% block content %}
+  <div class="govuk-grid-column-two-thirds">
+      <div id="google_translate_element" class="govuk-!-margin-bottom-6"></div>
+      <h1 class="govuk-heading-xl"> Find a legal aid adviser or family mediator </h1>
+      <p class="govuk-body">You need to provide information about your finances to find out if you qualify for legal aid.</p>
+      <p class="govuk-body"><a class="govuk-link" href="https://www.gov.uk/check-legal-aid" target="_blank" rel="noopener">Check if you qualify for legal aid (opens in a new tab)</a></p>
+  </div>
 
-      {% set errorMessage = {'err': ""} %}
+  {% block search_form %}
+      <div class="govuk-grid-column-full" style="background-color: #f3f2f1; padding: 1em; margin: 0.5em;">
```

Comment:
Inline CSS needs to be removed. This should be a CSS class that can be re-used throughout the project.

Reply:
@psweeting1 so my understanding of `gov.uk` classes, is that you can't override a base class. So if I made another class, it won't apply to the `gov.uk` base class. I can only amend using `gov.uk override classes` or as I've done above, add in additional stylings.